install/operator/pkg/generator/assets/addons/jaeger/server_operator.yml.tmpl

```diff
@@ -19,6 +19,12 @@
       metadata:
         labels:
           name: jaeger-operator
+{{if .Productized}}
+          com.redhat.component-name: jaeger-operator
+          com.redhat.component-version: "{{.Tag}}"
+          com.redhat.product-name: "Red_Hat_Fuse"
```

Comment:
Really think we need to discuss the difference between product and upstream as putting these into upstream just seem wrong.
Appreciate opinions please @zregvart @Delawen @heiko-braun @lgarciaaco @.....